workflow-job

```diff
@@ -448,7 +467,12 @@ flows["${sha1}"] = {
 
                 if (packagageOnGitlab()) {
                     wrap([$class: 'MaskPasswordsBuildWrapper', varPasswordPairs: [[password: "${gitlabToken}", var: 'gitlabToken']]]) {
-                        sh "curl -X POST -Ftoken=${gitlabToken} -F'ref=master' -F'variables[REPO_NAME]=oncommit' -F'variables[SOURCE_REPO]=https://github.com/linagora/james-project' https://ci.linagora.com/api/v4/projects/803/trigger/pipeline"
+                        if (isMaster()) {
+                            sh "curl -X POST -Ftoken=${gitlabToken} -F'ref=master' -F'variables[REPO_NAME]=oncommit' -F'variables[SOURCE_REPO]=https://github.com/linagora/james-project' https://ci.linagora.com/api/v4/projects/803/trigger/pipeline"
+                        }
+                        if (isTag()) {
+                            sh "curl -X POST -Ftoken=${gitlabToken} -F'ref=master' -F'variables[REPO_NAME]=oncommit' -F'variables[SOURCE_REPO]=https://github.com/linagora/james-project' -F'variables[TAG]=${tagName} https://ci.linagora.com/api/v4/projects/803/trigger/pipeline"
```

Comment:
this should not be on the oncommit REPO, but on the jamesxxx (where xxx is the openpaas version). Could be tricky to compute this...